src/components/layout/ConfirmAntCreation/ConfirmAntCreation.tsx

```diff
@@ -0,0 +1,46 @@
+import { ANTContractJSON } from '../../../types';
+import { NAME_PRICE_INFO } from '../../../utils/constants';
+import { AntCard } from '../../cards';
+import ArPrice from '../ArPrice/ArPrice';
+import { Tooltip } from '../Tooltip/Tooltip';
+
+function ConfirmAntCreation({ state }: { state: ANTContractJSON }) {
+  return (
+    <div className="register-name-modal center">
+      <AntCard
+        domain={''}
+        id={undefined}
+        state={state}
+        compact={false}
+        enableActions={false}
+        showTier={false}
+        overrides={{
+          targetId: state.records['@'].transactionId,
+        }}
+        disabledKeys={[
+          'tier',
+          'evolve',
+          'maxSubdomains',
+          'ttlSeconds',
+          'id',
+          'domain',
+          'leaseDuration',
+        ]}
+      />
+      <span className="text faded underline" style={{ maxWidth: '475px' }}>
+        You will sign a single transaction to deploy this ANT.
+      </span>
+      <div className="flex flex-column center" style={{ gap: '0.2em' }}>
+        <Tooltip message={NAME_PRICE_INFO}>
+          <span className="white bold text-small">
+            <ArPrice data={state} />
+            &nbsp;AR&nbsp;
```

Comment:
why not have the component do this?

Reply:
I think the ArPrice component is useful - in the future we can abstract auto update logic to it. Keeps the confirm component clean i think.

Reply:
yeah i mean have the `ArPrice` component return the full string `XX AR` instead of just the number

Reply:
Ah, gotchya. Hm. Yeah I suppose that works.